packages/mdadm/Cargo.lock

```diff
@@ -0,0 +1,131 @@
+# This file is automatically @generated by Cargo.
+# It is not intended for manual editing.
+version = 3
```

Comment:
remove this `Cargo.lock`

Reply:
Without this, `cargo make -e PACKAGE=mdadm build-package` fails.  What am I missing?

Reply:
Good question, and I too have this problem with `cargo make build-package`. It's not just you, and it's not just this package. I'll let you know as soon as I have an answer. Or, possibly, @bcressey will answer us both before I figure it out, as he does.

Reply:
Good news: thanks to @yeazelm I know when we introduced this misbehavior in our build system. The somewhat good news is that there is a workaround: if you build a variant that includes this package (e.g., you add it to the packages in the `aws-dev` variant and build that variant), the build system will happily build your package without a `Cargo.lock` file, as it builds the variant you have selected. I will see what we can do about making `cargo make build-package` work as intended, and update this pull request.